game/game.go

```diff
@@ -376,3 +365,43 @@ func (g *Game) Update(delta float64) {
 		}
 	}
 }
+
+// SetGameServerProperties reads cfg.Game.Server.* and overrides the default player
+// properties with values in the configuration json if set
+// TODO: There must be a better way to do this?
+func SetGameServerProperties(cfg *config.Config) {
+	if cfg.Game.Server.VerticalWall != nil {
```

Comment:
Happy to refactor this in the future if there is a better way to do this. I was unable to find one as I did this.